Sources/Build/Buildable.swift

```diff
@@ -59,16 +59,24 @@ extension Module: Buildable {
     }
 }
 
-extension SwiftModule {
-    func pkgConfigArgs() throws -> [String] {
-        return try recursiveDependencies.flatMap { module -> [String] in
+extension ModuleProtocol {
+
+    func pkgConfigArgsCombined() throws -> [String] {
```

Comment:
Cool will do that.

> It isn't really clear what this function conceptually maps to. It is passing Xcc (escaped for swiftc) + Xld, but Xld isn't escaped for swiftc. Abstractly, how can that be used correctly?
> 
> What probably happens is that Xld frequently includes arguments which are ok to pass straight to swiftc, but nothing is auditing that (instead, the user is just going to get weird errors from swiftc if a package tries to pass other linker flags).

Xld arguments should be escaped using `-Xlinker` for swiftc?